generator.yaml

```diff
@@ -110,6 +113,25 @@ resources:
         from:
           operation: DescribeProcessingJob
           path: FailureReason 
+  HyperParameterTuningJob:
+    exceptions:
+      errors:
+          404:
+            code: ResourceNotFound
+            message_prefix: Amazon SageMaker can't find a tuning job
+    fields:
+      HyperParameterTuningJobStatus:
```

Comment:
I believe there are other useful fields which should be populated under status and are not part of Spec. one such field is bestTrainingJob, its model location etc Check old operators for such fields